js/noms/src/value-store.js

```diff
@@ -53,36 +53,44 @@ export default class ValueStore {
   }
 
   // TODO: This should return Promise<?Value>
-  async readValue(hash: Hash): Promise<any> {
+  readValue(hash: Hash): Promise<any> {
     const entry = this._valueCache.entry(hash);
```

Comment:
Seems like this should be using `this._valueCache.get`, since the entry isn't used. Then again maybe `Cache.get` isn't actually used anywhere and you should delete that.